components/compliance-service/integration_test/reporting_server_list_suggestions_test.go

```diff
@@ -0,0 +1,206 @@
+package integration_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	iam_v2 "github.com/chef/automate/api/interservice/authz/v2"
+	authzConstants "github.com/chef/automate/components/authz-service/constants/v2"
+	"github.com/chef/automate/components/compliance-service/api/reporting"
+	reportingServer "github.com/chef/automate/components/compliance-service/api/reporting/server"
+	"github.com/chef/automate/components/compliance-service/ingest/events/compliance"
+	"github.com/chef/automate/components/compliance-service/reporting/relaxting"
+	rules_tags "github.com/chef/automate/lib/authz"
+)
+
+func ingestReport(fileName string, f func(*compliance.Report)) error {
```

Comment:
This is a nice tool for testing! I see moving this into the suit once other tests start using it.

Reply:
At first, I was wondering why we want to pass the func instead of just doing the work of the func right in this function, then I realized how flexible this is in our ability to change the report to our liking from the calling side! 
Nice!